addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -178,6 +180,9 @@ const Wysiwyg = Widget.extend({
             this.odooEditor.document.addEventListener("keyup", this._signalOnline, true);
         }
 
+        if (!this._value) {
+            this._value = this.getValue();
+        }
```

Comment:
Then here:
```js
this.setValue(this.getValue());
```
Would there be cases that don't work in that way ? If not then I think it would be more idiomatic.

Reply:
When we instantiate the wysiwyg for an Odoo FieldHtml : 
1. The `$editable` is empty.
2. the `options.value` is set and contain the value to initiate the wysiwyg with.

When we instantiate the wysiwyg for a website page :
1. The  `$editable` is already in the DOM and contain content.
2. the `options.value` is undefined

So in the Odoo field case we need to use the `options.value`  and store it in `this._value` to be used for comparison in the `isDirty()` ( already "worked" before), for the website wee need to take existing content of `$editable` and store it in `this._value` otherwise we would never be able to compare the current editor value to the inital one in the `isDirty()`